app-backend/migrations/src_migrations/main/scala/134.scala

```diff
@@ -0,0 +1,10 @@
+import slick.jdbc.PostgresProfile.api._
+import com.liyaos.forklift.slick.SqlMigration
+
+object M134 {
+  RFMigrations.migrations = RFMigrations.migrations :+ SqlMigration(134)(List(
+    sqlu"""
```

Comment:
Are there any instances of duplicates in staging or production we have to worry about? Might be good to account for that in the migration otherwise I think this will fail.

Reply:
good point